decomp/vis/ontology.py

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
I'm a bit worried about having a separate file listing attribute identifiers, because it's an extra bit of maintenance every time we push a new version of UDS that we are likely to forget. This ontology seems to me like it should be obtained from the corpus itself and that the corpus should produce these identifiers by aggregating across all nodes and edges in all graphs.

Reply:
I've changed this to infer the ontologies from the dev split, so that the loading time isn't too long. This increases the overhead for loading a visualization slightly, but it's definitely easier to maintain.

Reply:
@esteng metadata-handling has undergone a substantial upgrade: we now have explicit metadata objects defined in [`decomp.semantics.uds.metadata`](https://github.com/decompositional-semantics-initiative/decomp/blob/master/decomp/semantics/uds/metadata.py) that track both the typing of a property and the annotators that annotated it. Can you look into using these metadata objects in lieu of relying on inference? I'm hoping it won't be too heavy of a lift.